server/coordinator/coordinator_request.go

```diff
@@ -8,13 +8,95 @@ import (
 	"github.com/suborbital/appspec/request"
 	"github.com/suborbital/appspec/tenant"
 	"github.com/suborbital/appspec/tenant/executable"
-	"github.com/suborbital/deltav/scheduler"
-	"github.com/suborbital/deltav/server/coordinator/sequence"
+	"github.com/suborbital/e2core/scheduler"
+	"github.com/suborbital/e2core/server/coordinator/sequence"
 	"github.com/suborbital/vektor/vk"
 )
 
-func (c *Coordinator) vkHandlerForModule(mod tenant.Module) vk.HandlerFunc {
+func (c *Coordinator) vkHandlerForModuleByName() vk.HandlerFunc {
 	return func(r *http.Request, ctx *vk.Ctx) (interface{}, error) {
+		ident := ctx.Params.ByName("ident")
+		namespace := ctx.Params.ByName("namespace")
+		name := ctx.Params.ByName("name")
+
+		tnt := c.syncer.TenantOverview(ident)
+		if tnt == nil {
+			return nil, vk.E(http.StatusNotFound, "tenant not found")
+		}
+
+		var mod *tenant.Module
+
+		for i, m := range tnt.Config.Modules {
+			if m.Namespace == namespace && m.Name == name {
+				mod = &tnt.Config.Modules[i]
+				break
+			}
+		}
+
+		if mod == nil {
+			return nil, vk.E(http.StatusNotFound, "module not found")
+		}
+
+		req, err := request.FromVKRequest(r, ctx)
+		if err != nil {
+			ctx.Log.Error(errors.Wrap(err, "failed to request.FromVKRequest"))
+			return nil, vk.E(http.StatusInternalServerError, "failed to handle request")
+		}
+
+		req.UseSuborbitalHeaders(r, ctx)
```

Comment:
If this returns an error, it gets eaten, which would be bad. Looking into the method, there's a lot that can go wrong in there.